src/app/home/home.component.scss

```diff
@@ -115,6 +115,10 @@
             flex: 0.9;
             background-color: #00BFA5;
             border-radius: 20px;
+            .paymentFeed {
```

Comment:
naming of the class here is confusing - each mat card is taking styles from "paymentFeed", maybe change it to paymentCard or something like that

Reply:
and the other style names should be adjusted too